build.gradle

```diff
@@ -221,9 +223,10 @@ project('segmentstore:contracts') {
 project('segmentstore:storage') {
     dependencies {
         compile project(':common')
-        compile project(':shared')
+        compile project(':shared:protocol')
         compile project(':segmentstore:contracts')
         compile project(':shared:metrics')
+        compile group: 'org.apache.curator', name: 'curator-framework', version: apacheCuratorVersion
```

Comment:
Where is this being used in `storage`? I did a quick search and it's not.

Reply:
It is being used in BookKeeperLog and BookKeeperLogFactory